nuget/semver/major

```diff
@@ -0,0 +1 @@
+0
```

Comment:
what is this? did you mean to include it?

Reply:
Files in this folder are part of release automation and handle versioning.